### DISCUSSION TITLE
Sliding in of elements

### DISCUSSION THREAD

**Discussion by christoph-heinrich** [tomasklaen/uosc#189] [Ideas]
With the timeline growing as the mouse cursor approaches, it kind of looks like the timeline is sliding in from outside the screen.
It would be interesting how it looks to do have the volume slider or top bar slide in from outside as the mouse cursor approaches instead of (or in addition to) fading in.

Are you open to such a change?

**Comment by tomasklaen** [comment]
I did try to make all elements slide in once...

The reason only timeline slides is that it has the whole minimized feature, and also a very predictable and huge hitbox.

Everything else sliding is just annoying, because when it does, you never know where the hitbox is going to end up, so it's a moving target for your cursor. It was a very bad UX. Timeline is the only element where this is not that big of a deal, but I wouldn't even be sliding that if it didn't need the minimized state.

**Reply by christoph-heinrich** [reply]
I see... so that means if the element is attached to the border it shouldn't be a problem, which is the case for the top bar. Volume slider would also have to be attached to the border.

Maybe one day I'll whip up a poc just to try it out, but since you already did that once, it'll probably not go anywhere.
